packages/sdk/react-client/src/client/useAgentHostingProvider.ts

```diff
@@ -11,7 +11,14 @@ import { type Config } from '@dxos/config';
 import { log } from '@dxos/log';
 
 export const useAgentHostingProviderClient = (config: Config): AgentHostingProviderClient | null => {
```

Comment:
Since this is being exported from a core sdk package, can we add a jsdoc here which notes this as experimental in some way? one way we've done this is just by marking it as deprecated

Reply:
yeah, wasn't quite sure how to capture that 1) It's experimental and the internal and external API is unstable 2) It's a feature we're developing internally and not making the backend or infrastructure open source, 3) it's coupled to a product we're developing, which is why the client is kind of a road to nowhere, but doesn't preclude you from running your own agent, or even building a hosting platform of your own that matches the API

Reply:
ahh, I had forgotten about that aspect